### PR TITLE
Triage workflow update + Contributor guide update

### PR DESCRIPTION
**Pull request by jpvajda**
This PR:

1. removes the add `triage-needed` label from the workflow
2. Adds logic to put new PR in `ToDo` column on the project board.
3. updates the contributor guide to be more explicit with how you create a new installPlan
4. Explains the PR review workflow at a high level.

### CODE REVIEW

File: .github/workflows/pr-project-board.yml [hunk 1]
```diff
@@ -20,3 +21,4 @@ jobs:
       - name: Move PR to Project Board for triage
         run: |
           gh pr edit ${{github.event.pull_request.number}} --add-project "NR1 Quickstarts Community"
+          COLUMN=$(gh api -H "$HEADER" projects/$PROJECT_ID/columns --jq ".[] | select(.name == \"$COLUMN_NAME\").id")
```
Comment (nr-kkenney) [inline .github/workflows/pr-project-board.yml:24]: i think we will want to use the whole code block from the other workflow:
```sh
        PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
        HEADER="Accept: application/vnd.github.inertia-preview+json"
        PR_ID=$(gh api -H "$HEADER" "repos/$REPOSITORY/pulls?state=open" --jq ".[] | select(.number == $PR_NUMBER).id")
        PROJECT=$(gh api -H "$HEADER" "repos/$REPOSITORY/projects" --jq ".[] | select(.id == $PROJECT_ID).id")
        COLUMN=$(gh api -H "$HEADER" "projects/$PROJECT/columns" --jq ".[] | select(.name == \"$COLUMN_NAME\").id")
        gh api -H "$HEADER" -X POST "projects/columns/$COLUMN/cards" -f content_type='PullRequest' -F content_id="$PR_ID"
```
and remove `gh pr edit . . .`

File: .github/workflows/pr-project-board.yml [hunk 2]
```diff
@@ -19,4 +20,4 @@ jobs:
 
       - name: Move PR to Project Board for triage
         run: |
-          gh pr edit ${{github.event.pull_request.number}} --add-project "NR1 Quickstarts Community"
+          PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH") HEADER="Accept: application/vnd.github.inertia-preview+json" PR_ID=$(gh api -H "$HEADER" "repos/$REPOSITORY/pulls?state=open" --jq ".[] | select(.number == $PR_NUMBER).id") PROJECT=$(gh api -H "$HEADER" "repos/$REPOSITORY/projects" --jq ".[] | select(.id == $PROJECT_ID).id") COLUMN=$(gh api -H "$HEADER" "projects/$PROJECT/columns" --jq ".[] | select(.name == \"$COLUMN_NAME\").id") gh api -H "$HEADER" -X POST "projects/columns/$COLUMN/cards" -f content_type='PullRequest' -F content_id="$PR_ID"
```
Comment (rudouglas) [inline .github/workflows/pr-project-board.yml:23]: Would it make more sense to use board automation for this, might be a lot cleaner and easier to manage:
Screenshot 10-12-2021 13-51-39

**Comment by jpvajda** [comment]
@rudouglas ah-ha! yes, I'll use that. Thanks!

**Review by rudouglas — APPROVED** [review]
🎨

**Pull request merged**